src/components/faveList.js

```diff
@@ -0,0 +1,1054 @@
+import { useState } from "react";
+import {
+  ImageList,
+  ImageListItem,
+  ImageListItemBar,
+  // ImageListItemText,
+  // ImageListItemImage,
+  // ImageListItemSecondaryAction,
+  IconButton,
+} from "@mui/material";
+import {
+  Favorite,
+  HeartBroken,
+  StarBorder,
+  StarRate,
+} from "@mui/icons-material";
+
+import PlacePhoto from "./utility/PlacePhoto";
+
+function srcset(image, width, height, rows = 1, cols = 1) {
+  console.log(image);
+  return {
+    src: `${image}&w=${width * cols}&h=${height * rows}&fit=crop&auto=format`,
+    srcSet: `${image}&w=${width * cols}&h=${
+      height * rows
+    }&fit=crop&auto=format&dpr=2 2x`,
+  };
+}
+
+export default function CustomImageList() {
+  const [starred, setStarred] = useState(false);
+  return (
+    <ImageList
+      sx={{
+        width: "75%",
+        height: "60%",
+        // Promote the list into its own layer in Chrome. This costs memory, but helps keeping high FPS.
+        transform: "translateZ(0)",
+        borderRadius: "0.5rem",
+      }}
+      rowHeight={200}
+      gap={10}
+      // variant="quilted"
+      // variant="standard"
+      varinant="masonry"
+    >
+      {itemData.map((item) => {
+        const cols = item.featured ? 2 : 1;
```

Comment:
if this is the same value you do not need 2 vars

Reply:
I think it will be good to have it because according to the documentation you can "assign" column or row sizes to the images. I believe this can be received from the data according to the aspect ratio of the image.